apps/sms/js/thread_ui.js

```diff
@@ -158,7 +158,9 @@ var ThreadUI = {
     );
```

Comment:
also maybe just take out all the Threads.currentId/Threads.active information from threads.js.